### PR TITLE
[Feat] Implemented Chain ID & added required routing

### PR DESCRIPTION
**Pull request by a-singh09**
To access the dashboard of a reactor with address 0xABC123 deployed on a chain with Id, let's say, 2001 (milkomeda), we would use the URL -> https://gluon.stability.nexus/g#2001#0xABC123

### CODE REVIEW

File: src/app/(root)/create/page.tsx [hunk 1]
```diff
@@ -19,6 +19,17 @@ export default function Dashboard() {
                 { placeholder: "Symbol" },
                 { placeholder: "Token Address (existing ERC-20 token)" },
               ]}
+              dropdowns={[
+                {
+                  placeholder: "Select Chains",
+                  options: [
+                    { value: "1", label: "Ethereum Mainnet" },
```
Comment (ceilican) [inline src/app/(root)/create/page.tsx:26]: Let's just say "Ethereum" instead of "Ethereum Mainnet".

File: src/components/prompt-dialog-box.tsx [hunk 2]
```diff
@@ -42,12 +50,24 @@ export default function PromptDialogBox({ children }: PromptDialogBoxProps) {
               Please enter the contract address of the stablecoin you want to use.
             </AlertDialogDescription>
           </AlertDialogHeader>
+          <Select onValueChange={(value) => setChainId(value)}>
```
Comment (ceilican) [inline src/components/prompt-dialog-box.tsx:53]: I think the information about chainId and their names is duplicated in a few places. I would suggest creating a separate file with an array of chainId and chainName pairs, and importing this file/array in all the places where it is needed. Then when we need to add more chains, we just need to include more chains in one place.

**Review by ceilican — CHANGES_REQUESTED** [review]

File: src/app/(root)/create/page.tsx (hunk 1 above)
Comment (a-singh09) [reply src/app/(root)/create/page.tsx]: Sure, I have made the change.

File: src/components/prompt-dialog-box.tsx (hunk 2 above)
Comment (a-singh09) [reply src/components/prompt-dialog-box.tsx:54]: Sure, I have made a separate file for storing chainId and chainName pairs.

**Review by ceilican — APPROVED** [review]

**Pull request merged**